.env.example

```
@@ -48,6 +48,11 @@ DOCMIND_OLLAMA_ENABLE_LOGPROBS=false
 DOCMIND_OLLAMA_TOP_LOGPROBS=0
 DOCMIND_LMSTUDIO_BASE_URL=http://localhost:1234/v1
 DOCMIND_VLLM__VLLM_BASE_URL=http://localhost:8000
+# llama.cpp server (OpenAI-compatible GGUF runtime).
+# Use `llama-server --alias <name>` and set DOCMIND_MODEL to that alias when
+# DOCMIND_LLM_BACKEND=llamacpp.
+DOCMIND_LLAMACPP_BASE_URL=http://localhost:8080/v1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DOCMIND_LLAMACPP_BASE_URL key should go before the DOCMIND_LMSTUDIO_BASE_URL key

(UnorderedKey)